civitai-downloader-v2/convert_to_csv.py

```
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+"""Convert processed JSONL to CSV format."""
+
+import json
+import csv
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)